test/parallel/test-http-parser.js

```diff
@@ -104,6 +106,7 @@ function expectBody(expected) {
   assert.throws(function() {
     parser.execute(request, 0, request.length);
   }, Error, 'hello world');
+ */
```

Comment:
Does it pass now?

If I remember it right it just killed the process with uncaught exception last time (`FatalException` or whatever).